routes/quoteCalc.js

```diff
@@ -1,9 +1,28 @@
 const express = require("express");
 const router = express.Router();
 
-//Below is for get, change get to post for post request
-router.get("api url", (req, res) => {
-  //Api code
+function calculatePremium(car_value, risk_rating) {
```

Comment:
Why is this function duplicated here from api folder?

Reply:
Thanks for spotting this - I originally worked on the backend locally in a different structure to what had been set-up in GitHub, duplication likely happened due to my getting confused while transferring files over. The calculatePremium function is needed in quoteCalc.js in /routes, but I have removed it from the file in /api.